eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+    "root": true,
+    "extends": "airbnb-base",
+    "env": {
+      "node": true,
+      "es6": true,
+      "mocha": true
+    },
+    "rules": {
```

Comment:
Where are all these rules coming from?

Reply:
I was sent a script at the last boot-camp with this rules

Reply:
should i remove the rules?

Reply:
Remove the rules. Kindly check the product specification and do as it says. Check it very well

Reply:
ok